templates/awsebscsiprovisioner.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: kubeaddons.mesosphere.io/v1alpha1
 kind: Addon
 metadata:
   name: awsebscsiprovisioner
-  namespace: kubeaddons
+  namespace: kube-system
```

Comment:
why do we move it to kube-system ?

Reply:
This is done to maintain backwards compatibility:

Going forward the Addon will deploy to the namespace in the Addon metadata, as opposed to what's in the spec.

I would assert that changing which namespace the underlying Addon components are deployed to (as opposed to the Addon resource itself) is out of the scope of this PR.

Reply:
Correct!